Add config report generation to enterprise (#499)
owncloud/docs-server: modules/admin_manual/pages/enterprise/reporting/config_report.adoc

= Generate a Config Report
:toc: right

> :description: Generating a config report when using an Enterprise version does not differ compared when using the Community Edition. Therefore you can reuse that description.

Reviewer: ```suggestion
:description: Generating a config report when using an Enterprise version does not differ from the Community Edition.
```
